src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -64,17 +65,16 @@ export class NativeDateAdapter extends DateAdapter<Date> {
    * the result. (e.g. in the en-US locale `new Date(1800, 7, 14).toLocaleDateString()`
    * will produce `'8/13/1800'`.
    */
-  useUtcForDisplay: boolean;
+  useUtcForDisplay: boolean = true;
```

Comment:
Optional: can be simplified by making the `platform` a property and changing it to `useUtcForDisplay = !this.platform.TRIDENT`. This approach is easier to follow though.